src/main/java/com/qudini/jpa/Querying.java

```diff
@@ -45,14 +43,14 @@
      */
     @Nonnull
     public static <RootModel, Result> TypedQuery<Result> query(
-            final EntityManager entityManager,
-            final Class<RootModel> rootModel,
-            final Class<Result> resultModel,
-            final QueryAction<RootModel, Result> query
+            EntityManager entityManager,
+            Class<RootModel> rootModel,
+            Class<Result> resultModel,
+            QueryAction<RootModel, Result> query
```

Comment:
While `final` is important for fields, it isn't useful for locals and parameters (except for final variables without initialisers, in which case it triggers useful control flow analysis to ensure it is assigned only once). As other parts of this file wasn't using them, I removed them here for consistently.

Reply:
I'd prefer them to be consistently mandated, vote?